sso/provider/src/main/java/org/eclipse/kapua/sso/provider/SingleSignOnUtils.java

```diff
@@ -0,0 +1,112 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2020 Eurotech and/or its affiliates and others
```

Comment:
hey @gbarbon, if this file is new, header should contain only year "2020" and not 2019.

Reply:
Yep, that's true! I think this happened because I renamed the old `JsonUtils.java` class to `SingleSignOnUtils`. Anyway, now it's ok.